http/server.lua

```diff
@@ -23,6 +23,17 @@ local function sprintf(fmt, ...)
     return string.format(fmt, ...)
 end
 
+local function is_no_keepalive(disallowed_map, useragent)
+    if type(disallowed_map) ~= 'table' or useragent == nil then
```

Comment:
I guess we could check that "disallowed_map" on initialization step. And here two variants are possible - nil or table.
It seems to be hot path and I believe nil check is faster than type check.

Reply:
```lua
@@ -1299,6 +1299,11 @@ local exports = {
         if type(options) ~= 'table' then
             errorf("options must be table not '%s'", type(options))
         end
+        options.keepalive_disable = options.keepalive_disable or {}
+        if type(options.keepalive_disable) ~= 'table' then
+            error('Option keepalive_disable must be a table.')
+        end
+
         local default = {
             max_header_size     = 4096,
             header_timeout      = 100,

```